api/src/main/java/org/openmrs/Allergies.java

```diff
@@ -46,36 +46,36 @@ public boolean add(Allergy allergy) {
 		allergyStatus = SEE_LIST;
 		return allergies.add(allergy);
 	}
-	
+
 	public boolean remove(Allergy allergy) {
 		boolean result = allergies.remove(allergy);
 		if (allergies.isEmpty()) {
 			allergyStatus = UNKNOWN;
 		}
 		return result;
 	}
-	
+
 	@Override
 	public void clear() {
 		allergyStatus = UNKNOWN;
 		allergies.clear();
 	}
-	
-	public void confirmNoKnownAllergies() throws APIException {
```

Comment:
Why did you remove the throws clause?

Reply:
I did this because an APIException is a RuntimeException which is already automatically thrown, hence making the throws specification superfluous. 

Reference: https://sonar.openmrs.org/project/issues?id=org.openmrs%3Aopenmrs&resolved=false

Reply:
Cool, true that.